Geometry_Engine/Query/Clone.cs

```diff
@@ -136,6 +136,13 @@ public static NurbsSurface Clone(this NurbsSurface surface)
 
         /***************************************************/
 
+        public static PlanarSurface Clone(this PlanarSurface surface)
+        {
+            return new PlanarSurface(surface.ExternalBoundary, surface.InternalBoundaries.ToList());
```

Comment:
Think the boundaries, internal and external, should be cloned here as well. That has at least been the general approach for the rest of the close methods.

Reply:
Oh, yeah, that makes sense